optscale-deploy/optscale/templates/ngui.yaml

```diff
@@ -58,6 +58,20 @@ spec:
         env:
         - name: IMAGE_ID
           value: {{ $config.image.id }}
+        - name: UI_BUILD_PATH
+          value: /usr/src/app/ui
+        - name: PROXY_URL
+          value: https://ngingress-nginx-ingress-controller:443
```

Comment:
This variable is temporary, there is a fallback value in the server code, should work without setting it up here, we do not want to keep this in the overlay. This is a default ingress endpoint.

Reply:
will remove